vendor/k8s.io/kube-openapi/pkg/handler/handler.go

```diff
@@ -178,8 +179,8 @@ func (o *OpenAPIService) RegisterOpenAPIVersionedService(servePath string, handl
 							return
 						}
 					}
-					contentType := accepts.Type + "/" + accepts.SubType
-					w.Header().Set("Content-Type", contentType)
+					// contentType := accepts.Type + "/" + accepts.SubType
+					// w.Header().Set("Content-Type", contentType)
```

Comment:
@apelisse @Jefftree commenting this out made tests pass again... something about the content type being set is making the server choke. I think it does not like the @ symbol in the proto subtype, based on the error message coming from here:

https://cs.opensource.google/go/go/+/release-branch.go1.19:src/mime/mediatype.go;l=103-122

https://cs.opensource.google/go/go/+/release-branch.go1.19:src/mime/grammar.go;l=11-32;drc=119f679a3bd2e60cfc990920f82fd1a5cb006f4c

`@` seems to be disallowed as a token character by that library